### ISSUE TITLE
Riddley

### ISSUE THREAD

**Issue by KingMob** [clj-commons/meta#69]
Any thoughts on adding Riddley? Zach Tellman's long gone, but has generally been open to handing off stuff to clj-commons. It's not super-popular itself, but it's depended on by libs like aleph, manifold, potemkin, cloverage, and specter.

I ask because there's a handful of PRs with bug fixes that looks like they'd be useful to incorporate for manifold's `let-flow`.

**Comment by seancorfield** [comment]
Do we have a volunteer to maintain it?

**Comment by KingMob** [comment]
Me. I'm loath to take on more, but it shouldn't be too bad, since it's largely complete.

**Comment by seancorfield** [comment]
Yeah, I figured you were already maintaining some of the libs that depend on this.

So it's a "yes" from me, based on that. @danielcompton @slipset ?

**Comment by slipset** [comment]
Yes

**Comment by seancorfield** [comment]
That's (at least) two yeses, so can someone reach out to Zach about this one to kick off the transfer?

**Comment by seancorfield** [comment]
@slipset has reached out to Zach, who has agreed to transfer Riddley to clj-commons. We can keep this open until the transfer happens.

**Comment by borkdude** [comment]
+1 for riddley under clj-commons. I'm trying to make it babashka compatible and having that merged would enable new libs!

**Comment by borkdude** [comment]
My use case for riddley in bb is this:

```
/tmp/bb -Sdeps '{:paths ["src" "test"] :deps {cloverage/cloverage {:git/url "https://github.com/borkdude/cloverage" :git/sha "8f4a755fe7a1fcef65258539ecf76427df9b4520" :deps/root "cloverage"}}}' -m cloverage.coverage --help
```

It's cloverage, which uses riddley, running under babashka. Specter which also works with bb also uses riddley but I worked around that in specter. Would be nice to have that fully working too.

**Comment by seancorfield** [comment]
Riddley has been adopted into clj-commons.
